rllib/agents/dqn/apex.py

```diff
@@ -128,6 +148,54 @@ def __call__(self, item: Tuple[ActorHandle, SampleBatchType]):
 
 
 class ApexTrainer(DQNTrainer):
+    @override(Trainable)
```

Comment:
I really like the new structure of Trainers:
* setup() -> Create necessary algo components, e.g. replay buffers
* training_iteration() -> Calls different sub-methods for sampling, training, weight-synching, etc..

Looks great